src/Lykke.Quintessence.Core.Abstractions/Blockchain/Strategies/IBuildRawTransactionStrategy.cs

```diff
@@ -0,0 +1,9 @@
+namespace Lykke.Quintessence.Core.Blockchain.Strategies
+{
+    public interface IBuildRawTransactionStrategy
```

Comment:
Not necessary to fix but:
A way to detailed interface, Icon does not use such interface

Reply:
Open an Issue, please. Will fix it in next release (for BIL 2.0)